packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-text-like/gux-form-field-text-like.tsx

```diff
@@ -105,6 +115,11 @@ export class GuxFormFieldTextLike {
   @State()
   private hasHelp: boolean = false;
 
+  @State()
+  private wasExceeding: boolean = false;
+
+  @Event() characterLimitChange: EventEmitter<boolean>;
```

Comment:
From our call the last day I reduced the event emissions to just a single event which which will take the payload of whether the characters are exceeding or not which is a boolean value. Also maybe this should just be a generic error event potentially instead of being specific ?

I investigated other options but I do not think they are good options but let me know if you think they would be better. The first was to apply an attribute to the slotted input eg `data-character-limit-change=true|false` and the attribute would change based on the state internally. I just thought maybe by doing this it would be intrusive as we would be changing the consumers DOM. 

Another option maybe is to apply it to the host element.